cobigen/index.asciidoc

```diff
@@ -1,27 +1,23 @@
-= CobiGen Cli
+= Tutorial for Java Backend Generation with CobiGen
 ====
-CobiGen is a generic incremental code generator. It allows you build Java CRUD application based on the devonfw architecture including all software layers. You can generate all necessary classes and services (DAOs, Transfer Objects, simple CRUD use cases with REST services) of the application from one single entity class.
+CobiGen is a generic incremental code generator. It allows you to build Java CRUD application based on the devonfw architecture including all software layers. You can generate all necessary classes and services (DAOs, Transfer Objects, simple CRUD use cases with REST services) of the application from one single entity class.
 
 CobiGen provides Maven and Eclipse integrations. But there is also a command line interface (CLI) which enables the generation of code using only few commands. In this scenario, you will learn how to install and use the cobigen cli inside a java project.
```

Comment:
In this tutorial we use do not focus on the installation and the cli. So the last sentence is not needed here I think.